packages/pigeon/platform_tests/test_plugin/windows/pigeon/core_tests.gen.cpp

```diff
@@ -25,6 +25,13 @@ using flutter::EncodableList;
 using flutter::EncodableMap;
 using flutter::EncodableValue;
 
+FlutterError CreateConnectionError(const std::string channel_name) {
```

Comment:
This should either be `static` or in a `namespace { ... }` (i.e., an anonymous namespace).